docs/assessment/index.md

```diff
@@ -105,16 +153,56 @@ An assessment is broken down in to a list of zones, like this:
+| `title`         | string  | The title of the zone. (Optional; default: none)                                                                                                      |
+| `questions`     | array   | The list of slots for questions and question alternatives within the zone. (Optional; default: none)                                                  |
+| `numberChoose`  | integer | Number of questions to select for each student from this zone. (Optional; default: select all)                                                        |
+| `maxPoints`     | number  | Limit on the number of points that can be earned from this zone. (Optional; default: sum of question max points)                                      |
+| `bestQuestions` | integer | Only this many questions in the zone will count towards the total points (highest-point questions will count). (Optional; default: use all questions) |
 
-- Within each zone the question order is randomized based on [the assessment type and the `shuffleQuestions` option](#changing-question-order-randomization).
+Zone specification details are in the [format specification for `infoAssessment.json`](../schemas/infoAssessment.md)
 
-- An assessment question can be specified by either a single `id` or by a list of alternatives, in which case one or more of these alternatives is chosen at random. Once the question `id` is determined, then a random variant of that question is selected. Question alternatives inherit the points of their parent group, if specified.
+- Each zone appears in the given order in the assessment. Zone titles are optional and are displayed to the student if present.
+
+- Within each zone the question order is randomized based on [the assessment type and the `shuffleQuestions` option](#question-order-randomization).
 
 - If a zone has `maxPoints`, then, of the points that are awarded for answering questions in this zone, at most `maxPoints` will count toward the total points.
 
 - If a zone has `bestQuestions`, then, of the questions in this zone, only `bestQuestions` with the highest number of awarded points will count toward the total points.
 
+## Slots for questions and question alternatives
+
+Each zone has a list of _slots_ given by the `questions` array. Each slot contains either a single question `id`:
+
+```json
+{ "id": "hardQV1", "points": 10 }
+```
+
+Or a slot can contain a _question alternative list_:
+
+```json
+{
+  "numberChoose": 1,
+  "points": 5,
+  "alternatives": [{ "id": "FirstAltQ" }, { "id": "SecondAltQ" }]
+}
+```
+
+- An assessment question can be specified by either a single `id` or by a list of alternatives, in which case one or more of these alternatives is chosen at random. Once the question `id` is determined, then a random variant of that question is selected. Question alternatives inherit the points of their parent group, if specified.
```

Comment:
This is the second single-item list I'm seeing; is that intentional for some reason?

Reply:
Nope, just a result of updating the PR and splitting up a list into multiple parts or deleting all but one item for some reason. Fixed in de53592c59b594e38c8e036f35a362a26d176f9f